zero/gkyl_mat.h

```diff
@@ -345,6 +349,43 @@ gkyl_nmat_mem *gkyl_nmat_linsolve_lu_cu_dev_new(size_t num, size_t nrow);
  */
 void gkyl_nmat_linsolve_lu_release(gkyl_nmat_mem *mem);
 
+/**
+ * Allocate memory needed in modal to nodal conversion matrices needed 
+ * by cublasDgemm
+ *
+ * @param nr Number of rows in the matrix A
+ * @param nc Number of columns in the matrix A
+ * @param alpha Coefficient infront of A*B
+ * @param beta Coefficient infron of C
+ * @param transa Whether or not to transpose A
+ * @param transb Whether or not to transpose B
+ * @param use_gpu 
+ * @return Preallocated memory
+ */
+gkyl_mat_mm_array_mem *gkyl_mat_mm_array_mem_dev_new(int nr, int nc, double alpha, 
```

Comment:
All device allocation calls should be "cu_dev_new" not just "dev_new"

Reply:
Actually this memory is now both used by the CPU/GPU routines and the memory lives on one or the other, so I believe it should just be: gkyl_mat_mm_array_mem_new

Reply:
That's fine!